25_exceptions_lab/02.py

```diff
@@ -4,4 +4,14 @@
 
 Alert the user politely if there was any problem opening the file
 """
+import sys
 
+file = sys.argv[1]
+
+try:
+    with open(file, "rb") as fin:
```

Comment:
When opening the file as "binary" you should use the binary read operations (`f.read(bufsize)`). 

When reading lines open the file as text (omitting the "b")